content/applications/general/voip/onsip.rst

```diff
@@ -5,94 +5,141 @@ Use VoIP services in Odoo with OnSIP
 Introduction
 ============
 
-Odoo VoIP can be set up to work together with OnSIP (www.onsip.com). In that case, the installation
-and setup of an Asterisk server is not necessary as the whole infrastructure is hosted and managed
-by OnSIP.
-
-You will need to open an account with OnSIP to use this service. Before doing so, make sure that
-your area and the areas you wish to call are covered by the service. After opening an OnSIP account,
-follow the configuration procedure below.
+Odoo VoIP can be set up to work together with `OnSIP <https://www.onsip.com/>`_. An account is
+needed with OnSIP to use this service. Before doing so, make sure that the company's home area and
+the areas that will be called to are covered by OnSIP services. After opening an OnSIP account,
+follow the configuration procedure below to configure on an Odoo database.
 
 Configuration
 =============
 
-Go to Apps and install the module **VoIP OnSIP**.
+First, go to Apps and install the module *VoIP OnSIP*.
+
+.. tip::
+   Clear out the search bar and type in `onsip`. The OnSIP module will appear.
 
-.. image:: onsip/onsip01.png
+.. image:: onsip/install-onsip.png
    :align: center
+   :alt: View of Onsip app in the app search results.
+
+Odoo VoIP setting
+-----------------
 
-Go to Settings/General Settings. In the section Integrations/Asterisk (VoIP), fill in the 3 fields:
+Following the module installation, go to :menuselection:`Settings --> General Settings`. In the
+:menuselection:`Integrations -- > Asterisk (VoIP)` section, fill in the 3 fields:
 
-- **OnSIP Domain** is the domain you chose when creating an account on www.onsip.com. If you don't
-  know it, log in to https://admin.onsip.com/ and you will see it in the top right corner of the
-  screen.
-- **WebSocket** should contain wss://edge.sip.onsip.com
-- **Mode** should be Production
+- :guilabel:`OnSIP Domain` is the domain that was assigned when creating an account on
+  `OnSIP <https://www.onsip.com/>`_.
+- :guilabel:`WebSocket` = `wss://edge.sip.onsip.com`
+- :guilabel:`Mode` = :guilabel:`Production`
 
-.. image:: onsip/onsip02.png
+.. image:: onsip/asterisk-setting.png
    :align: center
+   :alt: VoIP configuration settings in Odoo Settings app.
+
+.. tip::
+   To access the OnSIP domain, navigate to `OnSIP <https://www.onsip.com/>`_ and log in. An
```

Comment:
When I logged in, it took me to the call history dashboard instead. I just had to click on the Administrators link at the top-right of the page, and then the rest of the steps were correct.